pkg/tfbridge/property_path.go

```diff
@@ -112,22 +127,166 @@ func propertyValueTriggersReplacement(
 	return replacement
 }
 
-// pathContainsComputed returns true if the schema contains a Computed property at a path prefixed by path.
-func pathContainsComputed(
-	path propertyPath, rootTFSchema shim.SchemaMap, rootPulumiSchema map[string]*info.Schema,
+// isPlanCompatibleWithInputs returns true if all values in the walkedValue are also in the comparedValue,
```

Comment:
I'll follow up with a refactor here to deduplicate some of the logic.

Reply:
This feels like a wrong file to put this function in.